README.md

```diff
@@ -31,13 +31,23 @@ The tool can be installed as a standalone command line tool. The following depen
 |Python|3.10|
 |VCFtools|0.1.16|
 
-### Install directly from GitHub
+Additionally, the `libmagic` C library must also be installed on the system.
+
+With the dependencies (and the proper versions) installed, install `pipeval` through one of the options below:
+
+### Install directly from GitHub through SSH
```

Comment:
Have you considered publishing to PyPI?

Reply:
We do want to publish it eventually to PyPI, it just hasn't been set up yet